docs/book/orders/cart-promotions.rst

```diff
@@ -62,6 +62,8 @@ The types of rules that are configured in **Sylius** by default are:
 * **Items From Taxon Total** - checks in the cart if items from a given taxon cost a given amount of money,
 * **Nth Order** - checks if this is for example the second order made by the customer,
 * **Shipping Country** - checks if the order's shipping address is in a given country.
+* **Customer Group** - checks if the current customer is in a given customer group.
+* **Contains Product** - checks if the order contains a certain product.
```

Comment:
Would be cool to sort rules the same way as in the admin panel

![image](https://github.com/Sylius/Sylius/assets/870747/496e42ea-f72e-4117-b759-de85a4e399c3)

Some of the rules no longer match admin panel (Not needed captial letters at each word for rest),
```patch
--- Taxon
--- Items From Taxon Total 
+++ Has at least one from taxons
+++ Total price of items from taxon
```

```
Cart quantity
Customer group
Nth order
Shipping country
Has at least one from taxons
Total price of items from taxon
Contains product
Item total
```

@zairigimad Hi, great catch :+1:  Please let me know if you would like to handle this, or I will do a follow up PR.

Reply:
Nice catch too @diimpp I can add it if you want in this PR since it’s not merged yet.

Reply:
That would be awesome, I'll raise this PR for faster merge afterwards.

Sorry for a long wait :pray:

Reply:
it's done.